### ISSUE TITLE
TabView.iosIconRenderingMode property

### ISSUE THREAD

**Issue by leocaseiro** [NativeScript/NativeScript#2689]
After accepting the #2435 and merging the c3ce128e6578271e31e03131f469cd78bd1186ea into core, the tab icons color have become always blue and we have to [harcoded](https://github.com/NativeScript/nativescript-angular/issues/395#issuecomment-239464667) to fix it back.

While we are not able to choose the icon tab color, I suggest we make optional to choose the render between `UIImageRenderingModeAutomatic` and `UIImageRenderingModeAlwaysOriginal`.

I'm happy to send a PR with something like:

`<TabView rendering="automatic">` or `<TabView rendering="alwaysoriginal">`

Other References Issues: #260 #1278

**Comment by DomGaud** [comment]
Update: With new version should be able to switch 
from **_automatic_** 
( `var originalRenderedImage = is.ios.imageWithRenderingMode(0);` )

to **_alwaysoriginal_**
 ( `var originalRenderedImage = is.ios.imageWithRenderingMode(1);` )

**Comment by hamorphis** [comment]
@leocaseiro @DomGaud So basically, if I add a new iOS-specific public property to the TabView class which allows setting the rendering mode everyone will be happy. Is that correct?

**Comment by hamorphis** [comment]
```
<TabView iosIconRenderingMode="alwaysOriginal">...</TabView>
```

**Comment by leocaseiro** [comment]
Hi @hamorphis, that's exactly the kind of solution I was expecting for. Perfect!

Thank you so much for that.

**Comment by DomGaud** [comment]
Perfect! Thanks @hamorphis

**Comment by hamorphis** [comment]
@leocaseiro @DomGaud The pull request is merged into the master branch. It will be released with 2.5.0. since we are already in a code-freeze for 2.4.0

**Comment by leocaseiro** [comment]
Thank you so much for that @hamorphis.

I really appreciate it!
